src/errorsHandlers/user.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+function userError(name, userId = 1) {
```

Comment:
Better check fields separately. Don't mix it in one utility. As I see - you've added default value for userId to avoid error if it's not provided - just separate logic for checking different fields.
You can create common utilities for checking numeric, textual fields etc. And then combine them in cases: `checkUserName(name)`, `checkUserData(name, userId)`.